tools/IoTEdgeDevOps/VstsPipelineSync/VstsBuildBatchUpdate.cs

```diff
@@ -32,9 +32,21 @@ public async Task RunAsync(TimeSpan waitPeriodAfterEachUpdate, CancellationToken
         {
             var buildManagement = new BuildManagement(devOpsAccessSetting);
             var releaseManagement = new ReleaseManagement(devOpsAccessSetting);
+            var bugManagement = new BugManagement(devOpsAccessSetting);
 
             while (!ct.IsCancellationRequested)
             {
+                try
+                {
+                    Console.WriteLine($"Import bugs data started at {DateTime.UtcNow}");
+
+                    await ImportVstsBugDataAsync(bugManagement);
+                }
+                catch (Exception ex)
+                {
+                    Console.WriteLine($"Unexcepted Exception: {ex}");
+                }
```

Comment:
There is similar logic below and this loop is getting long. We need to refactor this at some point. If you have any suggestions lmk.

Reply:
Remove try-catch here and move Console.WriteLine inside ImportVstsBugDataAsync and similar to others.